### PR TITLE
Fix for logging not being turned on

### PR DESCRIPTION
**Pull request by gfodor**
I'm not sure how or why I thought this was working before -- but it seemed to not be.

### CODE REVIEW

File: src/hub.html [hunk 1]
```diff
@@ -17,6 +17,13 @@
     <% } else { %>
         <script src="https://cdn.rawgit.com/aframevr/aframe/3e7a4b3/dist/aframe-master.js" integrity="sha384-EaMOuyBOi9ERV/lVDwQgz/yFWBDWPsIju5Co6oCZZHXuvbLBO81yPWn80q0BbBn3" crossorigin="anonymous"></script>
     <% } %>
+
+    <script>
+        // HACK this has to run after A-Frame but before our bundle, since A-Frame blows away the local storage setting
+        if (/[&?]debug=true/.test(document.location.search) && typeof(localStorage) !== 'undefined') {
```
Comment (cvan) [inline src/hub.html]: `typeof(localStorage)` -> `typeof localStorage`

Comment (cvan) [reply src/hub.html]: generally better to just use [`window.location`](https://www.w3.org/TR/html50/browsers.html#the-location-interface), as opposed to `document.location`

File: src/hub.html [hunk 2]
```diff
@@ -17,6 +17,13 @@
     <% } else { %>
         <script src="https://cdn.rawgit.com/aframevr/aframe/3e7a4b3/dist/aframe-master.js" integrity="sha384-EaMOuyBOi9ERV/lVDwQgz/yFWBDWPsIju5Co6oCZZHXuvbLBO81yPWn80q0BbBn3" crossorigin="anonymous"></script>
     <% } %>
+
+    <script>
+        // HACK this has to run after A-Frame but before our bundle, since A-Frame blows away the local storage setting
```
Comment (cvan) [inline src/hub.html]: `HACK` -> `HACK:`

File: src/hub.html (hunk 1 above)
Comment (cvan) [inline src/hub.html]: ' -> " (for prettier linting style)

File: src/hub.html (hunk 2 above)
Comment (cvan) [reply src/hub.html]: can you [file an A-Frame issue](https://github.com/aframevr/aframe/issues/new) to handle this?

File: src/hub.html (hunk 1 above)
Comment (gfodor) [reply src/hub.html]: interesting, why is that?

**Pull request merged**